outport/process/transactionsfee/transactionChecker.go

```diff
@@ -87,7 +87,11 @@ func isSCRWithRefundNoTx(scr *smartContractResult.SmartContractResult) bool {
 	return ok && differentHash && hasRefund
 }
 
-func isRelayedTx(tx *transactionWithResults) bool {
+func (tep *transactionsFeeProcessor) isRelayedTx(tx *transactionWithResults, epoch uint32) bool {
+	if tep.enableEpochsHandler.IsFlagEnabledInEpoch(common.RelayedTransactionsV1V2DisableFlag, epoch) {
```

Comment:
This function was not used on the Relayed V3 flow, correct (it seems not)?

Reply:
maybe rename this function to indicate V1V2